playbooks/customize.yml

```diff
@@ -28,6 +28,10 @@
       prompt: "Enable Shadowsocks? Press enter for default "
       default: "yes"
       private: no
+    - name: streisand_shadowsocks_v2ray_enabled
+      prompt: "Enable v2ray-plugin for Shadowsocks? Press enter for default "
```

Comment:
Perhaps we could add a note similar to:

```
Enable v2ray-plugin for Shadowsocks? (Note: v2ray helps bypass Deep Packet Inspection, but you must configure your client device to support it). Press enter for default 
```

Reply:
Thanks for the comment, but since streisand documentation already mentions the role and significance of `v2ray-plugin`, therefore we can safely assume that the users already know about it. Plus adding too many details in the prompt might make it look ugly.